### ISSUE TITLE
Not using CUDA GPU despite that being available

### ISSUE THREAD

**Issue by tongclement** [mvirgo/MLND-Capstone#37]
Will edit the code to add the capability for it to use CUDA GPU (As I am trying to use this as a benchmark)

**Comment by mvirgo** [comment]
It should be using CUDA GPU as a default; potentially if it isn't it may be because you have a CUDA version that doesn't match to the TensorFlow version in the environment used here.

**Comment by tongclement** [comment]
Yes unfortunately it seems like that this is the problem. A lot of projects on GitHub user TensorFlow V1 and it is really difficult to have multiple cuda versions for different projects. I have heard about docker and I will try it in the near future. Thanks for open sourcing your amazing project and replying anyways :)
